packages/cli-kit/src/public/node/ink.ts

```diff
@@ -1 +1 @@
-export {Box, Text, useInput, useStdin} from 'ink'
+export {Static, Box, Text, useInput, useStdin} from 'ink'
```

Comment:
@shauns Is it okay to export Static from here? I wasn't if there was a better way to use this component. I tried to setup something similar to what the [ConcurrentOutput ](https://github.com/Shopify/cli/blob/main/packages/cli-kit/src/private/node/ui/components/ConcurrentOutput.tsx#L192-L215)component uses.